core/trino-main/src/main/java/io/trino/execution/BasicStageStats.java

```diff
@@ -214,6 +236,11 @@ public long getRawInputPositions()
         return rawInputPositions;
     }
 
+    public DataSize getSpilledDataSize()
+    {
+        return spilledDataSize;
+    }
+
     public Duration getPhysicalInputReadTime()
```

Comment:
nit: this is out of place, should be below `getPhysicalInputPositions` (separate commit)